### DISCUSSION TITLE
峥果卷帘门控制器 zinguo.motor.mk01 电机控制应该设成按钮状

### DISCUSSION THREAD

**Discussion by MarcPolaris** [XiaoMi/ha_xiaomi_home#120] [Ideas]
峥果卷帘门控制器 zinguo.motor.mk01 电机控制应该设成按钮状，而非目前的下拉选项，太难用，参考[xiaomi-miot]集成那样才更方便，谢谢。
<img width="814" alt="截屏2024-12-17 22 33 19" src="https://github.com/user-attachments/assets/a0ffe07e-7b70-487a-8f4e-94fcf32b9c6d" />
<img width="929" alt="截屏2024-12-17 22 34 23" src="https://github.com/user-attachments/assets/fbf7ecc0-d7bc-4ab9-9f29-0f81d4fb3122" />

**Comment by SusanPhevos** [comment]
v0.3.1 已支持。
